jobbergate-cli/jobbergate_cli/end_to_end_testing/job_scripts.py

```diff
@@ -0,0 +1,75 @@
+import json
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List
+
+import buzz
+
+from jobbergate_cli.end_to_end_testing.applications import get_test_applications
+from jobbergate_cli.end_to_end_testing.base import BaseEntity
+from jobbergate_cli.end_to_end_testing.constants import JOB_SCRIPTS_CACHE_PATH
+from jobbergate_cli.end_to_end_testing.utils import cached_run
+
+
+def get_test_job_scripts() -> List[Path]:
+    return list(JOB_SCRIPTS_CACHE_PATH.glob("*.json"))
+
+
+@dataclass
+class JobScripts(BaseEntity):
+    base_applications: List = field(default_factory=get_test_applications)
+
+    def create(self):
+        for app in self.base_applications:
+            app_data = json.loads(app.read_text())
+            name = app_data["application_name"]
+            cached_run(
+                "create-job-script",
+                "--no-submit",
+                "--fast",
+                "--name",
+                name,
+                "--application-identifier",
+                app_data["application_identifier"],
+                "--description",
+                app_data["application_description"],
+                cache_path=JOB_SCRIPTS_CACHE_PATH / f"{name}.json",
+            )
+
+    def get(self):
+        for app in get_test_job_scripts():
+            app_data = json.loads(app.read_text())
+
+            id_value = str(app_data["id"])
+
+            result = cached_run("get-job-script", "--id", str(id_value))
+
+            with buzz.check_expressions(
+                "get-job-script returned unexpected data for --id={}".format(
```

Comment:
probably better to just use f-string.

Reply:
solved in [a990526](https://github.com/omnivector-solutions/jobbergate/pull/225/commits/a990526055f5f08f7dc732cce337cbc5fedb923e)